ml-agents/mlagents/trainers/learn.py

```diff
@@ -109,17 +108,8 @@ def run_training(run_seed: int, options: RunOptions) -> None:
             env_settings.env_args,
             os.path.abspath(run_logs_dir),  # Unity environment requires absolute path
         )
-        engine_config = EngineConfig(
-            width=engine_settings.width,
-            height=engine_settings.height,
-            quality_level=engine_settings.quality_level,
-            time_scale=engine_settings.time_scale,
-            target_frame_rate=engine_settings.target_frame_rate,
-            capture_frame_rate=engine_settings.capture_frame_rate,
-        )
-        env_manager = SubprocessEnvManager(
-            env_factory, engine_config, env_settings.num_envs
-        )
+
+        env_manager = SubprocessEnvManager(env_factory, options, env_settings.num_envs)
```

Comment:
The whole RunOptions are now sent to SubprocessEnvManager (and the subprocess worker)